scripts/add_electricity.py

```diff
@@ -827,12 +762,10 @@ def add_nice_carrier_names(n, config):
     Nyears = n.snapshot_weightings.objective.sum() / 8760.0
 
     # Snakemake imports:
-    regions = snakemake.input.regions
-
-    load_paths = snakemake.input["load"]
     countries = snakemake.config["countries"]
     admin_shapes = snakemake.input.gadm_shapes
     scale = snakemake.config["load_options"]["scale"]
```

Comment:
Please remove unnecessary entries... I think `scale` is not required here?
Please check the others as well.

Reply:
I removed them all except demand_profiles, since they were not needed